packages/gestalt/src/sharedSubcomponents/OverridingSpan.tsx

```diff
@@ -31,7 +37,7 @@ export default function OverridingSpan({
       : styles.textColorOverrideLight;
   }
   // Error type enforces bold weight
-  if (isError) {
+  if (isError && !isInExperiment) {
```

Comment:
I think we still need the override to enforce the color when we pass Text rather than string

Reply:
{ experiment ? styles.VRtextColorOverrideError : styles.textColorOverrideError }

Reply:
DONE